src/Settings.cpp

```diff
@@ -41,11 +41,9 @@ void Settings::setSettingsObject()
 
```

Comment:
Just for better performance, if you add a check in `setSettingsObject()` covering all the code as follows:

```
if(!settings) {
...
}
```
then we would avoid reserving the memory, constructing the object, parsing the file or the registry, and finally deleting the object. It should work perfectly, although it had to be checked. I think it's ready for merging and the optimization can be done later, but I say it if you want to try.

Reply:
Nice idea! I don't want anyone to be inconvenienced by my code. I'll fix it as soon as I get home. 😄

Reply:
I just fixed the code. (4e45bce33db86c9776bb9db8335fefbcf1c1b902)